unstructured_ingest/v2/processes/connectors/pinecone.py

```diff
@@ -121,19 +125,23 @@ def conform_dict(self, element_dict: dict) -> dict:
                 }
             )
 
+        metadata = flatten_dict(
+            pinecone_metadata,
+            separator="-",
+            flatten_lists=True,
+            remove_none=True,
+        )
+        metadata["file_id"] = file_data.identifier
```

Comment:
would it make sense to call this `record_id`?

Whatever we choose, we should be consistent across all connectors.

Reply:
Ya I wasn't sure what the best label for this was, we don't want to clash with anything that might already exist in the metadata being flattened. I'm cool with `record_id`.

Reply:
Created it as a constant in a new file that will be reused across other uploaders we add this functionality to.